test/jit/test_remove_mutation.py

```diff
@@ -2,40 +2,19 @@
 import sys
 
 import torch
-from torch.nn import functional as F
 from torch.testing import FileCheck
 
 # Make the helper files in test/ importable
 pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 sys.path.append(pytorch_test_dir)
-from torch.testing._internal.jit_utils import JitTestCase, freeze_rng_state
+from torch.testing._internal.jit_utils import JitTestCase
 
 if __name__ == '__main__':
     raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
                        "\tpython test/test_jit.py TESTNAME\n\n"
                        "instead.")
 
 class TestRemoveMutation(JitTestCase):
-    def test_lower_linear(self):
-        # linear is one of main use cases of removing mutation so add test so it doesnt regress
```

Comment:
Should another test be added to make sure the mutation removal still works?

Reply:
There is still `test_aten_inplace` down that checks mutation removal.